backend/infrahub/send/webhook.py

```diff
@@ -0,0 +1,37 @@
+from typing import Any
+
+import ujson
+from prefect import flow
+from prefect.logging import get_run_logger
+
+from infrahub.exceptions import NodeNotFoundError
+from infrahub.send.models import SendWebhookData
+from infrahub.services import services
+from infrahub.webhook import CustomWebhook, StandardWebhook, TransformWebhook, Webhook
+
+
+@flow(name="event-send-webhook")
```

Comment:
I think we need to define some naming conventions around the flows and tasks. I'd say that an "event" is something that happens within a system and based on that we perform some kind of "action". But it shouldn't block us now, just thinking ahead.